architecture/leather-architecture.d2

```diff
@@ -0,0 +1,155 @@
+vars: {
+  d2-config: {
+    layout-engine: dagre
+    theme-overrides: {
+      N1: '#12100F'
+      N2: '#6D625B'
+      N4: '#EAE6DF'
+      N5: '#CBC6BD'
+      N7: '#FFFFFF'
+      B1: '#12100F'
+      B2: '#433B33'
+      B3: '#DDD7CF'
+      B4: '#EAE6DF'
+      B5: '#F7F4F0'
+      B6: '#FFFFFF'
+      AA4: '#544D45'
+      AA5: '#726B63'
+      AB4: '#3B3530'
+      AB5: '#6D625B'
+    }
+  }
+}
+
+style.bold: false
+
+title: LEATHER ARCHITECTURE {
+  near: top-center
+  shape: text
+  style: {
+    font-size: 44
+    italic: true
+    font-color: '#12100F'
+  }
+}
+
+mono: {
+  label: ''
+  entity layer: {
```

Comment:
Don't think we need this extra "entity layer". Models already kind of implicitly define and form the core of the domain.

Reply:
I separated this just to make absolutely clear that at the core we have models that simply define the structure of the data 

Many (not all) onion style diagrams have entity at the centre, or divide domain services and models

![image](https://github.com/user-attachments/assets/ed54710d-1be9-47a1-98c2-cfbd8afce2a6)

![image](https://github.com/user-attachments/assets/e697f056-35cc-4f70-87b9-b2ea00101249)